public/index.html

```diff
@@ -3,19 +3,18 @@
     <head>
         <meta charset="utf-8">
         <title>Chat</title>
+        <!-- external libraries !-->
         <script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular.min.js"></script>
```

Comment:
Ah, actually you probably don't want to do that for another reason. Bundling them would stop them from being cached nicely as well. You would possibly do that in another case, but you can ignore it here.